puppet.gemspec

```diff
@@ -1,24 +1,24 @@
 Gem::Specification.new do |spec|
-  spec.name = "puppet"
+  spec.name = "openvox"
   spec.version = "8.11.0"
   spec.licenses = ['Apache-2.0']
 
   spec.required_rubygems_version = Gem::Requirement.new("> 1.3.1")
   spec.required_ruby_version = Gem::Requirement.new(">= 3.1.0")
-  spec.authors = ["Puppet Labs"]
+  spec.authors = ["Vox Pupuli"]
   spec.date = "2012-08-17"
```

Comment:
Our gemspec would be this year, wouldn't it?

Reply:
huh. I didn't even look at that tbh. I assumed that it was programmatically generated somewhere.

Reply:
(in any case, it's out of scope of this PR)